front/app/hooks/__mocks__/useLocalize.ts

```diff
@@ -0,0 +1 @@
+export default () => jest.fn((multiloc) => multiloc['en']);
```

Comment:
Excellent, thanks for adding additional mocks!

Reply:
My pleasure